tools/dependencies/known-dependencies.txt

```diff
@@ -437,6 +437,7 @@ jsr311-api-1.1.1.jar
 jvm-attach-api-1.5.jar
 kafka-clients-2.0.0.jar
 kafka-clients-2.4.1.jar
+kafka-clients-3.2.0.jar
```

Comment:
Can we unify the kafka-clients version? If you don't want to do this in this PR, you need to add announce in `LICENSE` file (NOTICE file it already contains this).
The same goes to below newly dependency.

Reply:
Thanks, done